src/components/card/archiveNoteCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import "./notecard.css";
+import UnarchiveIcon from "@mui/icons-material/Unarchive";
+import DeleteIcon from "@mui/icons-material/Delete";
+import {
+  deleteArchiveNoteHandler,
+  postRestoreArchiveHandler,
+} from "../../utils";
+import { useAuth } from "../../context/auth/auth-context";
+import { useData } from "../../context/data/data-context";
```

Comment:
use barrel export

Reply:
missed out on this one. thanks for pointing it out.